server/libs/infra/src/db/entities/user.entity.ts

```diff
@@ -49,4 +50,7 @@ export class UserEntity {
 
   @OneToMany(() => TagEntity, (tag) => tag.user)
   tags!: TagEntity[];
+
+  @OneToMany(() => AssetEntity, (asset) => asset.owner)
+  assets!: AssetEntity[];
```

Comment:
Do you need to run migration for this?

Reply:
I don't believe so. This is just adding the relation, which already exists in the database, to the entity so they it could be used in the code. So it is not actually changing anything in the database.